benchmarks/triton_kernels_benchmark/benchmark_testing.py

```diff
@@ -193,6 +194,36 @@ def perf_report(benchmarks):
     return wrapper
 
 
+@dataclass
+class MarkArgs:
+    reports: str = ""
+    n_runs: int = 1
+    parser: argparse.ArgumentParser = argparse.ArgumentParser()
+
+    @classmethod
+    def _parse_common_args(cls) -> tuple[argparse.Namespace, list[str]]:
```

Comment:
The second item in the tuple is not used?
```suggestion
    def _parse_common_args(cls) -> argparse.Namespace:
```

Reply:
It will be used in the entry point.